test_data/samples_test.go

```diff
@@ -135,12 +135,13 @@ func TestSimple(t *testing.T) {
 			}
 
 			mc := malcontent.Config{
-				Concurrency: runtime.NumCPU(),
-				IgnoreSelf:  false,
-				IgnoreTags:  []string{"harmless"},
-				Renderer:    simple,
-				RuleFS:      []fs.FS{rules.FS, thirdparty.FS},
-				ScanPaths:   []string{binPath},
+				Concurrency:           runtime.NumCPU(),
+				IgnoreSelf:            false,
+				IgnoreTags:            []string{"harmless"},
+				QuantityIncreasesRisk: true,
```

Comment:
@tstromberg -- without this, some of our known malicious samples were registering as `high` instead of `critical`, but for some reason it's not happening outside of this branch which is interesting.

Reply:
That said, we do use `QuantityIncreasesRisk: true` for the markdown tests but not the diff or JSON tests.